features/step_definitions/run_steps.rb

```diff
@@ -18,10 +18,10 @@
 end
 
 Then(/^warder detects( no)? (.+) (issues|violations)$/) do |no, what, _|
-  executing_output = send(:"executing_#{what.gsub(' ', '_')}")
+  executing_output = send(:"executing_#{what.tr(' ', '_')}")
   step "the output should#{' not' if no} contain \"#{executing_output}\""
 
-  validation_output = send(:"#{what.gsub(' ', '_')}_output")
+  validation_output = send(:"#{what.tr(' ', '_')}_output")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.